### ISSUE TITLE
percentage variable name is confusing

### ISSUE THREAD

**Issue by Akuli** [ThePhilgrim/minesweeper#33]
To me:
- A percentage means e.g. 25 for 25% mines.
- A mine count, or `how_many_mines_user_wants`, means how many mines there are, e.g. 200.

These are different things. But variable names are confusing, and `percentage` is a variable whose value is actually a mine count.

```
        self.how_many_mines_user_wants = percentage
```

Also, when there is an actual percentage, it is called `slider_value` instead. Even `real_percentage` isn't really a percentage; it's a mine count that hasn't been rounded to an integer yet.

```
    slider_value = int(difficulty_slider.scale.get())
    real_percentage = (width * height / 100) * slider_value
    percentage = round(real_percentage)
```

**Comment by ThePhilgrim** [comment]
Not completely sure what you mean, but I tried to clear it up a bit. Tell me if it needs further attention

**Comment by Akuli** [comment]
`real_percentage` and `percentage` distinction was fixed in ddcb96f6a675ed0913b1083e494eea994d039f0c but the rest of it is still just as confusing as before. If you have a variable whose value is the mine count, then why in the world does it have to be named `rounded_percentage` instead of `mine_count`?

**Comment by ThePhilgrim** [comment]
Fixed this in pr #40
